EventsExpress/DTO/UserInfo.cs

```diff
@@ -19,6 +19,7 @@ public class UserInfo
         public string Token { get; set; }
         public bool AfterEmailConfirmation { get; set; } = false;
         public IEnumerable<CategoryDto> Categories { get; set; }
+        public IEnumerable<RefreshToken> RefreshTokens { get; set; }
```

Comment:
Do we need this on UI app? We have already set Refresh token into cookie

Reply:
deleted